### PR TITLE
Add carousel field formatter

### PR DESCRIPTION
**Pull request by xendk**

### CODE REVIEW

**Comment by bigslott** [comment]
review ok -  der er noget udkommenteret kode, men det går jeg ud fra er meningen

File: profiles/ding2/libraries/slick/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.sass-cache
+node_modules
```
Comment (kallehauge) [inline profiles/ding2/libraries/slick/.gitignore]: Missing new line

File: sites/all/modules/breol/features/breol_news/breol_news.features.field_instance.inc [hunk 2]
```diff
@@ -170,14 +170,64 @@ function breol_news_field_default_field_instances() {
...
+      'type' => 'ting_search_carousel_widget',
+      'weight' => 5,
     ),
   );
 
   // Translatables
   // Included for use with string extractors like potx.
   t('Cover Image');
   t('Indhold');
+  t('Karouseller');
```
Comment (kallehauge) [inline sites/all/modules/breol/features/breol_news/breol_news.features.field_instance.inc]: I can see that line 229 is also in danish but is this in purpose?

File: sites/all/modules/reol_field_carousel/reol_field_carousel.module [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+/**
+ * Accecss callback for the ajax page.
+ *
+ * @param string $entity_type
+ *   The entity type the field is attached to.
+ * @param string $entity_id
+ *   The id of the entity the field is attached to.
+ * @param string $field_name
+ *   Name of the field.
+ */
```
Comment (kallehauge) [inline sites/all/modules/reol_field_carousel/reol_field_carousel.module]: If you're documenting parameters, shouldn't the `@return` also be documented?

**Comment by kallehauge** [comment]
A lot of functions doesn't have a description so it's implied that the user should know what is going on to read it. But the code itself looks good :+1:

**Pull request merged**